src/Main.java

```diff
@@ -1,5 +1,102 @@
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.Scanner;
+
 public class Main {
+    private static File currentDirectory;
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        Scanner scanner = new Scanner(System.in);
```

Comment:
File manager-related code should be extracted to a separate class, why did you put everything in `main`?